### PR TITLE
Add base for dockerfile

### PR DESCRIPTION
**Pull request by Hekku2**
Add docker file so this project can be compiled without having to install more stuff locally (this is done in preparation so I can verify those library updates etc.)

Also:
 * Update `npm-run-all` to 4.1.5 to avoid using event-stream@3.3.6 which is no longer available.
 * Update  `node-sass` to 4.12.0 so it works with newer node.
 * Fix Readme link to example page.
 * Remove `package-lock.json` in favor of `yarn.lock`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,9 +51,9 @@
     "codelyzer": "^2.0.1",
     "core-js": "^2.4.1",
     "css-loader": "^0.28.0",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "node-sass": "^4.5.2",
-    "npm-run-all": "^4.0.2",
+    "extract-text-webpack-plugin": "^2.1.2",
+    "node-sass": "4.12.0",
+    "npm-run-all": "4.1.5",
```
Comment (tarlepp) [inline package.json:56]: Within libraries like this we should not lock dependencies.

Comment (Hekku2) [reply package.json:56]: Good point.

BTW, do we have a need for both, `package-lock.json` and `yarn.lock` in this project?

**Comment by tarlepp** [comment]
Nope there should be just one of those


ke 4. elok. 2021 klo 11.11 Heikki-Jussi Niemi ***@***.***>
kirjoitti:

**Comment by Hekku2** [comment]
Okay, I'll choose yarn.lock then.

**Pull request merged**